components/more-less/more-less.js

```diff
@@ -201,11 +201,13 @@ class MoreLess extends LocalizeCoreElement(LitElement) {
 	__adjustToContent_makeInactive() {
 		this.inactive = true;
 		this.expanded = false;
-		this.__maxHeight = `${this.__content.scrollHeight}px`;
+		// Include 1 px of give room to account for issues with Firefox rounding the content's scroll height
```

Comment:
It would not be the first time we had to add a funky work-around for one particular browser. Unfortunate but this is what we might have to live with.

```suggestion
		// Include 1px of given room to account for issues with Firefox rounding the content's scroll height
```

Reply:
Oops, thanks for catching the typos. 😅

Reply:
> It would not be the first time we had to add a funky work-around for one particular browser. Unfortunate but this is what we might have to live with.

Yeah, that's fair. If Gaudi is comfortable with this change, I'm happy to merge it. I'm personally doubtful that there's a better solution given the browser APIs available to us, but I was holding out at least a sliver of hope that someone might know of something I missed. 😆